presentation/src/main/java/com/stop/TroubleShooterApplication.kt

```diff
@@ -0,0 +1,8 @@
+package com.stop
+
+import android.app.Application
+import dagger.hilt.android.HiltAndroidApp
+
+@HiltAndroidApp
+class TroubleShooterApplication : Application() {
+}
```

Comment:
```suggestion
class TroubleShooterApplication : Application()
```

클래스의 body가 비어있다면 braces를 생략할 수 있어요